frontend/packages/dev-console/src/components/health-checks/create-health-checks-probe-utils.ts

```diff
@@ -6,8 +6,8 @@ import { healthChecksDefaultValues } from './health-checks-probe-utils';
 
 export const constructProbeData = (data: HealthCheckProbeData, resourceType?: Resources) => {
   const probeData = {
-    ...(data.failureThreshold && { failureThreshold: data.failureThreshold }),
-    ...(data.successThreshold && { successThreshold: data.successThreshold }),
+    ...(data.failureThreshold && { failureThreshold: _.toInteger(data.failureThreshold) }),
```

Comment:
could we use `parseInt` here instead of lodash?

Reply:
There are some other `_.toInteger` calls in this function before. I think its good to use a similar way in one function.

Reply:
lodash is only being used for `toInteger` here which can be done using `parseInt`.
shouldn't we prefer using standard built in functions to external library?

Reply:
@nemesis09 You're waaay too late to that party lol. We have corrected it in a few places, but for the most part it is a mixed bag. There isn't too much loss to using lodash, and there is often a lot of code safety to gain in the utilities.

My preference is definitely to go with native functions when they do the same thing, to reduce our reliance on "magic" 3rd party utility functions. But in the end it doesn't really matter.